insertion_sort.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import unicode_literals
+from time import time
+
+
+def insertion_sort(init_list):
```

Comment:
Its a good idea to stay away from naming functions and files the same. It gets confusing real quick.